### PR TITLE
Error string should not be capitalized or end with punctuation.

### PR DESCRIPTION
**Pull request by m14815**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Error string should not be capitalized or end with punctuation.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
hi, thanks for the PR but error strings should end with space.
you can make them start with lowercase.

example bellow.

File: test/utils/create_resources.go [hunk 1]
```diff
@@ -55,182 +55,182 @@ func RetryWithExponentialBackOff(fn wait.ConditionFunc) error {
 
 func CreatePodWithRetries(c clientset.Interface, namespace string, obj *v1.Pod) error {
 	if obj == nil {
-		return fmt.Errorf("Object provided to create is empty")
+		return fmt.Errorf("Object provided to create is empty ")
```
Comment (neolit123) [inline test/utils/create_resources.go:58]: example:
```suggestion
		return fmt.Errorf("object provided to create is empty")
```

**Comment by m14815** [comment]
Ok, I will make some changes

**Review by neolit123 — COMMENTED** [review]
please squash the commits to one and force push.
i can then LGTM / approve.

/triage accepted
/ok-to-test

**Comment by m14815** [comment]
I think I did correctly, right?

**Review by neolit123 — COMMENTED** [review]
yes, thank you.

/lgtm
/approve

**Pull request merged**